index.js

```diff
@@ -1,82 +1,136 @@
-let repoList = []
+let repoList = [];
+const SEARCH_TERM = "-web-";
 
 function handleGetLabs() {
```

Comment:
If a user didn't define USER_ID and/or TOKEN_ID a ReferenceError: TOKEN_ID is not defined error will be thrown instead of hitting the else where a helpful message would be shown (unlike the way it was until now where getToken() & getUsername() defaulted to empty strings).

2 ways to go about it might be to either check over here for the presence of USER_ID and TOKEN_ID or have getToken() and getUsername() return something falsy if the consts aren't defined.